cpp/include/raft/random/detail/multi_variable_gaussian.cuh

```diff
@@ -286,5 +291,159 @@ class multi_variable_gaussian_impl {
   ~multi_variable_gaussian_impl() { deinit(); }
 };  // end of multi_variable_gaussian_impl
 
+enum class multi_variable_gaussian_decomposition_method { CHOLESKY, JACOBI, QR };
```

Comment:
Could we consider moving this to a file `random_types.hpp` or `multi_variable_gaussian_types.hpp` (though the former seems easier to find and use) for user visibility (and discoverability)? This also makes it easier for users to know what APIs are safe to expose through their public APIs.

Reply:
Done; thanks!